python/lsst/meas/algorithms/psfDeterminer.py

```diff
@@ -19,25 +19,56 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
-from lsst.pex.config import makeRegistry
-from .pcaPsfDeterminer import PcaPsfDeterminer
+from __future__ import absolute_import, division, print_function
 
-__all__ = ["psfDeterminerRegistry"]
+import abc
 
-psfDeterminerRegistry = makeRegistry(
-    '''A registry of PSF determiner factories
+import lsst.pipe.base as pipeBase
+import lsst.pex.config as pexConfig
 
-    A PSF determiner factory makes a class with the following API:
+__all__ = ["BasePsfDeterminerConfig", "BasePsfDeterminerTask", "psfDeterminerRegistry"]
 
-    def __init__(self, config, schema=None):
+class BasePsfDeterminerConfig(pexConfig.Config):
+    kernelSize = pexConfig.Field(
+        doc = "radius of the kernel to create, relative to the square root of the stellar quadrupole moments",
+        dtype = float,
+        default = 10.0,
+    )
+    kernelSizeMin = pexConfig.Field(
+        doc = "Minimum radius of the kernel",
+        dtype = int,
+        default = 25,
+    )
+    kernelSizeMax = pexConfig.Field(
+        doc = "Maximum radius of the kernel",
+        dtype = int,
+        default = 45,
+    )
+
+
+class BasePsfDeterminerTask(pipeBase.Task):
```

Comment:
I hate to ask, but could you change the commit message to have the new class name (BasePsfDeterminerTask) instead of the old one? I only ask because it confused me for a bit. Thanks.